chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/WorkloadAggregator.java

```diff
@@ -136,16 +136,7 @@ private void aggregateValues(String sessionId, String taskId, WorkloadTask workl
         double succeeded = (double) (invoked - failed);
         log.debug("Latency: avg {} stdev {}", avgLatency, stdDevLatency);
 
-        double throughput = Math.rint(succeeded / duration * 100) / 100;
-        if (Double.isNaN(throughput)) {
-            log.error("throughput is NaN (succeeded={},duration={}). Value for throughput will be set zero", succeeded, duration);
-            throughput = 0;
-        }
-        log.debug("Throughput: {}", throughput);
-
-        double successRate = (invoked.doubleValue() > 0)
-                ? Math.rint(succeeded / invoked.doubleValue() * 10000) / 10000
-                : 1;
+        double successRate = (invoked.doubleValue() > 0) ? Math.rint(succeeded / invoked.doubleValue() * 10000) / 10000 : 1;
```

Comment:
It's in DurationLogProcessor and it was here.
This code was just logging wrong throughput value, that's why it's deleted